pages/index.js

```diff
@@ -51,44 +75,53 @@ export default function Homepage({ trips, setTrips }) {
       </TravelLogLink>
       <Scrollbox>
         <StyledHeading2>Create a new Trip</StyledHeading2>
-        <TripInputForm handleAddTrip={handleAddTrip} />
-        <StyledHeading2>Current Trips</StyledHeading2>
-        {currentTrips.length === 0 ? (
-          <p>You do not have any current trips yet.</p>
+        <TripInputForm
+          handleAddTrip={handleAddTrip}
+          startDate={startDate}
+          endDate={endDate}
+          handleChange={handleChange}
+        />
+        {successMessage && <StyledHeading2>{successMessage}</StyledHeading2>}
+
+        <StyledHeading2>Upcoming Trips</StyledHeading2>
+        {upcomingTrips.length === 0 ? (
+          <p>There are no upcoming trips yet.</p>
         ) : (
-          currentTrips.map((trip) => (
+          upcomingTrips.map((trip) => (
             <StyledArticle key={trip.id}>
-              <StyledLink href="/explore">
+              <StyledLink href={`/${trip.slug}`}>
                 <StyledImageWrapper>
                   <StyledImage
                     src={trip.image}
                     height={800}
                     width={800}
-                    alt={trip.location}
+                    alt={trip.name}
                   />
                 </StyledImageWrapper>
-                <StyledHeading3>{trip.location}</StyledHeading3>
+                <StyledHeading3>
+                  {trip.name} - {countdown(trip.startDate)}
+                </StyledHeading3>
               </StyledLink>
             </StyledArticle>
           ))
         )}
-        <StyledHeading2>Upcoming Trips</StyledHeading2>
-        {upcomingTrips.length === 0 ? (
-          <p>You do not have any upcoming trips yet.</p>
+        <StyledHeading2>Current Trips</StyledHeading2>
```

Comment:
Usually we do not work with numbers as these can confuse (what is the reltation between the styled component and the number? Does this refer to the position, the amount of headings etc.) and we want to use names that describe better the purpose of the according element. `StyledTitle` might be a better name in this case.